### PR TITLE
docs(guides): fix typos on operator, sdk & submitting proofs

### PR DESCRIPTION
**Pull request by cliraa**
## Changes

* `docs/guides/0_submitting_proofs.md`: replace EigenlayerCLI for EigenLayer CLI
* `docs/guides/1_SDK.md`: minor typos
* `docs/operator_guides/0_running_an_operator.md`: replace Eth for ETH and Eigen for EigenLayer

### CODE REVIEW

**Review by entropidelic — APPROVED** [review]

File: docs/guides/0_submitting_proofs.md [hunk 1]
```diff
@@ -75,15 +75,15 @@ aligned deposit-to-batcher \
 --amount 0.1ether
 ```
 
-This commands allows the usage of the following flags:
+These commands allows the usage of the following flags:
```
Comment (glpecile) [inline docs/guides/0_submitting_proofs.md:78]:
```suggestion
These commands allows the usage of the following flags:

```

File: docs/guides/0_submitting_proofs.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ aligned get-user-balance \
 --user_addr <user_addr>
 ```
 
-This commands allows the usage of the following flags:
+These commands allows the usage of the following flags:
```
Comment (glpecile) [inline docs/guides/0_submitting_proofs.md:96]:
```suggestion
These commands allows the usage of the following flags:

```

File: docs/guides/1_SDK.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Aligned SDK
 
 The Aligned SDK aims to help developers interact with Aligned in a simple way.
-Some of its functionalities include submitting and verify proofs through the Aligned batcher, as well as checks the inclusion of the verified proofs on-chain. This guide provides an overview of the SDK, its installation, usage, and API details.
+Some of its functionalities include submitting and verifying proofs through the Aligned batcher, as well as checks the inclusion of the verified proofs on-chain. This guide provides an overview of the SDK, its installation, usage, and API details.
```
Comment (glpecile) [inline docs/guides/1_SDK.md]:
```suggestion
Some of its functionalities include submitting and verifying proofs through the Aligned Batcher, as well as checking the inclusion of the verified proofs on-chain. 

This guide provides an overview of the SDK, its installation, usage, and API details.
```

**Review by glpecile — CHANGES_REQUESTED** [review]

File: docs/guides/0_submitting_proofs.md (hunk 1 above)
Comment (cliraa) [reply docs/guides/0_submitting_proofs.md:78]: Done.

File: docs/guides/0_submitting_proofs.md (hunk 2 above)
Comment (cliraa) [reply docs/guides/0_submitting_proofs.md:96]: Done.

File: docs/guides/1_SDK.md (hunk 3 above)
Comment (cliraa) [reply docs/guides/1_SDK.md]: Done.

**Review by glpecile — APPROVED** [review]
LGTM 👍

**Pull request merged**